templates/data/slack-bot/slack-bot-build.yaml

```diff
@@ -0,0 +1,18 @@
+templates:
+  - from: generic.tmpl
+    render:
+      - to: ../../prow/jobs/slack-bot/slack-bot-build.yaml
+        jobConfigs:
+          - repoName: "github.com/kyma-incubator/slack-bot"
+            jobs:
+              - jobConfig:
+                  name: pull-slack-bot-build
+                  args:
+                    - --name=slack-bot
+                    - --context=./
+                    - --dockerfile=Dockerfile
+                    - --config=/config/kaniko-build-config.yaml
+                inheritedConfigs:
+                  global:
+                    - kaniko_buildpack
+                    - jobConfig_postsubmit
```

Comment:
Is it a presubmit (`pull-` in the name) or postsubmit? (`jobConfig_postsubmit`). OR do you want both jobs?